elodie/geolocation.py

```diff
@@ -161,9 +161,10 @@ def reverse_lookup(lat, lon):
 
     try:
         params = {'format': 'json', 'key': key, 'lat': lat, 'lon': lon}
+        headers = {"Accept-Language": "en-US,en;q=0.5"}
```

Comment:
Accepted-Language impacts the response. My system language is Russian and without this fix I get the place names in Russian, but Elodie doesn't work with non-ascii symbols. 
As you proposed,    Accepted-Language is moved to `constants.py`